client/state/jetpack-connection-health/selectors/get-jetpack-connection-health-request-error.js

```diff
@@ -0,0 +1,19 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns error code if the current site has possible Jetpack connection problem
+ * @param  {Object}  state         Global state tree
+ * @param  {number}  siteId        Site ID
+ * @returns {?string}              Connection error code
+ */
+export default function getJetpackConnectionHealthRequestError( state, siteId ) {
+	if ( ! siteId ) {
+		return null;
+	}
+	const siteState = state.jetpackConnectionHealth[ siteId ];
+	if ( ! siteState ) {
+		return null;
+	}
+
+	return state?.requestError;
```

Comment:
Did you mean:

```
return siteState?.requestError;
```

if yes, then we could shorten this to:

```suggestion
	return state.jetpackConnectionHealth[ siteId ]?.requestError ?? null;
```

that way we will also always return `null` in the case of non-error, instead of a mix of `null` and `undefined`.